opentreemap/api/instance.py

```diff
@@ -45,6 +45,24 @@ def wrapper(request, *args, **kwargs):
                  in instance_info_dict['field_key_groups']
                  if 'collection_udf_keys' not in field_group]
 
+        if request.api_version < 4:
+            multichoice_fields = {
+                field for field, info
+                in instance_info_dict['fields'].iteritems()
+                if info['data_type'] == 'multichoice'}
+
+            # Remove multichoice fields from perms
```

Comment:
I named it that because this list is created from walking over all of the `FieldPermission` rows for the users role.  I can change it.